### ISSUE TITLE
Update to latest NEXT-stable does not work minified

### ISSUE THREAD

**Issue by kylebyerly-hp** [grommet/grommet#2294]
Updating from
https://github.com/grommet/grommet.git#173fe04b37980287261aac6e32a1555d8974ad05
to
https://github.com/grommet/grommet.git#d2cbb769e2c13cbf9d0b8ac97a5aec0573d4c638 (also tried https://github.com/grommet/grommet.git#7012f96dd1f3dc3a2aff299d15c97d791f6a8ff6)
results in minified code stack tracing (but not development mode).
### Expected Behavior

New version should work both minified and not.

### Actual Behavior

New version does not work minified.

### URL, screen shot, or Codepen exhibiting the issue
Hard to reproduce with Codepen, as only happens minified.

Is the code that hits it,
```
// (C) Copyright 2017-2018 Hewlett Packard Enterprise Development LP

import { hpe } from 'grommet2';
import { deepMerge } from 'grommet2/utils/object';
import { colorForName } from 'grommet2/utils/colors';

export default deepMerge(hpe, {
  global: {
    colors: {
      'border': 'rgba(0, 0, 0, 0.15)',
      'Amazon': '#ff9801',
      'Azure': '#035bdb',
      'dark-4': '#666666',
      'NCS': hpe.global.colors.brand,
    },
    size: {
      'sidebar-wide': '480px',
      'sidebar-narrow': '240px',
    },
    drop: {
      backgroundColor: {
        light: '#ffffff',
        dark: '#222222',
      },
      border: {
        width: '12px',
        radius: '12px',
      },
    },
  },
  button: {
    border: {
      radius: '24px',
    },
  },
  icon: {
    color: colorForName('brand', hpe),
  },
});
```

 and here is the source-mapped stack trace.

```
Uncaught TypeError: Cannot read property 'global' of undefined
    at Object.<anonymous> (theme.js:14)
    at r (bootstrap:19)
    at Object.<anonymous> (index.js:10)
    at r (bootstrap:19)
    at Object.<anonymous> (index.js:4)
    at r (bootstrap:19)
    at Object.<anonymous> (index.js:9)
    at r (bootstrap:19)
    at Object.<anonymous> (index.js:85)
    at r (bootstrap:19)

```

### Steps to Reproduce
 1. 
 2. 
 3. 

### Your Environment

* Grommet version:
* Browser Name and version: All
* Operating System and version (desktop or mobile):

**Comment by atanasster** [comment]
Hi did you upgrade styled-components to version 4 and grommet-icons to 3.1-beta.

Just shooting in the dark, the stack trace doesnt seem very informative

**Comment by alansouzati** [comment]
investigating this now

**Comment by alansouzati** [comment]
How did you find the latest styled-components v4 minified?

**Comment by alansouzati** [comment]
One thing I've started to notice is that libraries are starting to drop support for minified javascript given their preference for the module system.

Can you expand why you are using Grommet minified? It is rather big since it includes all components and icons.

We included in the early days because of codepen, but now we have codesandbox. I was thinking about dropping support for the minified js as well.

**Comment by kylebyerly-hp** [comment]
Not using minified, but webpack's `loaderOptionsConfig.minimize = true;`

**Comment by kylebyerly-hp** [comment]
Even with 4.0.0-beta for styled-components and 3.1.0-beta grommet-icons the error still exists when packaged via webpack.

**Comment by alansouzati** [comment]
here is a repo 
[grommet-webpack-bug.zip](https://github.com/grommet/grommet/files/2399138/grommet-webpack-bug.zip)

it is an ejected create react app. can you try to edit `config/webpack.config.prod.js` to reproduce this problem?

create react app uses a rather old version of webpack `"webpack": "2.5.1",`. 

But you can run `npm run build` and then `server -s build`, using the code you provided it works just fine. I have a feeling this is not a grommet isssue.

In our sample apps we are using webpack 4 (they have a better support for tree shaking and they auto minify the code for production).

I'm closing this issue for now. But let's keep this conversation going. if we find the bug, we can always reopen it.

**Comment by florianbepunkt** [comment]
Experiencing the same issue. I use create-react-app. Interestingly everything worked, until I upgraded my deps

grommet from 2.0.0-beta.6 to 2.0.0-rc
grommet-icons from 3.0.0 to 3.1.0
react from 16.5.2 to 16.6.0
react-dom from 16.5.2 to 16.6.0
react-scripts from 2.0.3 to 2.0.5

I did not use an ejected webpack config. Can also confirm that this has to do with
`import { hpe } from "grommet/themes";`
